trends_view/index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
```

Comment:
This seems to be good. Could you please provide a couple of lines in README.md regarding running this to see how it works.